bootstrap/011_warehouse_schedules.sql

```diff
@@ -10,23 +10,36 @@ CREATE OR REPLACE PROCEDURE INTERNAL.UPDATE_WAREHOUSE_SCHEDULES(last_run timesta
 DECLARE
     task_outcome variant default (select object_construct());
 BEGIN
+    -- Get the configured timezone or default to 'America/Los_Angeles'
+    let tz text;
+    call internal.get_config('default_timezone') into :tz;
+    if (tz is null) then
+        tz := 'America/Los_Angeles';
+    end if;
+    task_outcome := (select object_insert(:task_outcome, 'opscenter timezone', :tz));
+    task_outcome := (select object_insert(:task_outcome, 'account timezone', internal.get_current_timezone()));
+
     -- The task calls this procedure with NULL and lets the procedure figure out the details.
     -- The ability to specify timestamps is only to enable testing.
     if (this_run is NULL) then
-        this_run := (select current_timestamp());
+        this_run := (select CONVERT_TIMEZONE(internal.get_current_timezone(), :tz, current_timestamp()));
```

Comment:
whats the behaviour of this when we pass in an LTZ timezone instead of an NTZ?

Reply:
For the 3-arg CONVERT_TIMEZONE function, I don't believe there is any difference if you pass an LTZ or an NTZ (because the timestamp is assumed to be in the timezone specified by the first argument).

I _think_ changing this procedure to accept an NTZ instead of an LTZ is the way we solve the goofiness of the timestamps shifting when given to a procedure. I had been messing around with this over the weekend which works as I expect:
```
create or replace procedure internal.test_tz(now timestamp_ntz)
returns table(tz_ts TIMESTAMP_NTZ, ts timestamp_ltz, tz_ts2 TIMESTAMP_NTZ, ts2 TIMESTAMP_NTZ)
language sql
as
begin
    let res resultset := (
        select convert_timezone(internal.get_Current_timezone(), 'America/New_York', current_timestamp()),
                current_timestamp(),
                convert_timezone(internal.get_current_timezone(), 'America/New_York', :now),
                :now);
    return table(res);
end;
```